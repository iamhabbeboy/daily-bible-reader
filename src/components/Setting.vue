<template>
  <div class="px-0 my-20 mx-auto w-8/12 text-right ">
    <h4 class="text-lg text-white">
      Setting
      <font-awesome-icon :icon="['fa', 'cog']" />
    </h4>
  </div>
</template>
