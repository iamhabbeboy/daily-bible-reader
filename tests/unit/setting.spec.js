import { shallowMount } from "@vue/test-utils";
import { FontAwesomeIcon } from "@fortawesome/vue-fontawesome";
import Setting from "@/components/Setting";

describe("Setting.vue", () => {
  //   beforeEach(() => {
  //     localVue = createLocalVue();
  //     localVue.use(library);
  //   });
  //   it("should take snapshot", () => {
  //     const wrapper = shallowMount(Setting);
  //     expect(wrapper).toMatchSnapshot();
  //   });

  it("should register font awesome icon", () => {
    // const wrapper = shallowMount(Setting);
    // const fontAwesome = FontAwesomeIcon();
    const wrapper = shallowMount(Setting, {
      FontAwesomeIcon
    });
    expect(wrapper).toMatchSnapshot();
  });
});
