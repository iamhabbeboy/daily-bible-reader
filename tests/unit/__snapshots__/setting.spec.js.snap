// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Setting should take snapshot 1`] = `
<div class="px-0 my-20 mx-auto w-8/12 text-right ">
  <h4 class="text-lg text-white">
    Setting
    <font-awesome-icon icon="fa,cog"></font-awesome-icon>
  </h4>
</div>
`;

exports[`Setting.vue should register font awesome icon 1`] = `
<div class="px-0 my-20 mx-auto w-8/12 text-right ">
  <h4 class="text-lg text-white">
    Setting
    <font-awesome-icon icon="fa,cog"></font-awesome-icon>
  </h4>
</div>
`;

exports[`Setting.vue should take snapshot 1`] = `
<div class="px-0 my-20 mx-auto w-8/12 text-right ">
  <h4 class="text-lg text-white">
    Setting
    <font-awesome-icon icon="fa,cog"></font-awesome-icon>
  </h4>
</div>
`;
